journal/profiles.cpp

```diff
@@ -49,11 +49,25 @@ T DeserializePointer(Player::PointerMap const& pointer_map,
   }
 }
 
```

Comment:
Add a comment saying that the following two functions use `std::string`s to store non-UTF-8 (UTF-16 specifically) because proto is silly and uses `std::string` for `bytes` as well as `string`.

Reply:
Done.